src/test/java/de/tum/in/www1/artemis/StudentExamIntegrationTest.java

```diff
@@ -1108,7 +1111,9 @@ private void assertVersionedSubmission(Submission submission) {
             assertThat(((TextSubmission) submission).getText()).isEqualTo(versionedSubmission.get().getContent());
         }
         else if (submission instanceof ModelingSubmission) {
-            assertThat(((ModelingSubmission) submission).getModel()).isEqualTo(versionedSubmission.get().getContent());
+            ModelingSubmission modelingSubmission = (ModelingSubmission) submission;
+            assertThat("Model: " + modelingSubmission.getModel() + "; Explanation: " + modelingSubmission.getExplanationText())
+                .isEqualTo(versionedSubmission.get().getContent());
```

Comment:
```suggestion
                    .isEqualTo(versionedSubmission.get().getContent());
```
That should fix one of the style errors.

Reply:
Should be resolved.